lib/PrincipleStudios.OpenApiCodegen.Client.TypeScript.Test/Utils/CommonDirectoryFixture.cs

```diff
@@ -48,13 +48,13 @@ private async Task Initialize()
         }
 
         // Ensure our common package was built
-        var tscResult = await NodeUtility.Tsc(psi => psi.WorkingDirectory = SolutionConfiguration.TypeScriptPackagePath, CancellationToken);
+        var tscResult = await NodeUtility.TscOpenApiCodegenTypeScriptPackage(psi => psi.WorkingDirectory = SolutionConfiguration.TypeScriptPackagePath, CancellationToken);
         if (tscResult.ExitCode != 0) throw new InvalidOperationException("tsc failed!") { Data = { ["error"] = tscResult.Error } };
 
         await WritePackageJson();
         await WriteTsconfigJson();
 
-        var exitCode = await NodeUtility.NpmInstall(SetupProcess, CancellationToken);
+        var exitCode = await NodeUtility.NpmInstall("typescript@latest", SetupProcess, CancellationToken);
```

Comment:
`npx tsc` doesn't work locally without `typescript` being installed - rather than picking a specific version, I figured we should always just check the latest.